packages/block-library/src/pullquote/index.js

```diff
@@ -62,8 +64,8 @@ export const settings = {
 	attributes: blockAttributes,
 
 	styles: [
-		{ name: 'default', label: __( 'Regular' ), isDefault: true },
-		{ name: SOLID_COLOR_STYLE_NAME, label: __( 'Solid Color' ) },
+		{ name: SOLID_COLOR_STYLE_NAME, label: __( 'Solid Color' ), isDefault: true },
```

Comment:
For similarity with the border style name, should we rename this to `Solid` in the UI?

Reply:
The change was applied, thank you for the suggestion.